app/forms/supplier_user_form.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class SupplierUserForm
+  include ActiveModel::Model
+  include ActiveModel::Serialization
+
+  attr_accessor :full_name, :email, :supplier
+  validates :supplier, presence: { message: "Select one" }, on: :supplier
+  validates :full_name, presence: { message: "Enter a name" }, on: :details
+  validates :email, presence: { message: "Enter email" }, on: :details
+  validate :email_not_taken, on: :details
+
+  def attributes
+    { "full_name" => nil, "email" => nil, "supplier" => nil }
```

Comment:
We can probably use the new ruby way of creating hashes 
```
{ full_name: nil, ... }
```

Reply:
I want the keys to be strings, not symbols

Reply:
Just curious why you specifically want strings.

Reply:
Misunderstood the [docs](https://api.rubyonrails.org/classes/ActiveModel/Serialization.html). They had strings, so I figured that's how the underlying methods worked